File: RSPAClient/Core/AtPar.ViewModel/VM_POU_CREDIT_HEADER.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_POU_CREDIT_HEADER
    {
        public long TRANSACTION_ID { get; set; }
        public string PATIENT_ID { get; set; }
        public string ACCOUNT_ID { get; set; }
        public string EXAM_ID { get; set; }
        public string COMMENTS { get; set; }
        public string DATE_OF_SERVICE { get; set; }
        public string PATIENTID_CHANGED { get; set; }
        public string DATE_TIME { get; set; }
        public string PATIENT_NAME { get; set; }
        public string CAPTURE_DATE_TIME { get; set; }
        public string USER_ID { get; set; }
        public bool REVIEWED { get; set; }
        public string DEPARTMENT_ID { get; set; }

    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_POU_CREDIT_HEADER.ts
import { VM_POU_CREDIT_DETAILS } from './VM_POU_CREDIT_DETAILS';
export class VM_POU_CREDIT_HEADER {
    public TRANSACTION_ID: number;
    public PATIENT_ID: string;
    public ACCOUNT_ID: string;
    public EXAM_ID: string;
    public COMMENTS: string;
    public DATE_OF_SERVICE: string;
    public PATIENTID_CHANGED: string;
    public DATE_TIME: string;
    public PATIENT_NAME: string;
    public CAPTURE_DATE_TIME: string;
    public USER_ID: string;
    public REVIEWED: boolean;
    public DEPARTMENT_ID: string;
    public DETAILS: VM_POU_CREDIT_DETAILS[] = [];
}